scss/_base_media.scss

```diff
@@ -49,4 +49,8 @@
   [hidden] {
     display: none;
   }
+
+  .p-media-container {
```

Comment:
This should not be part of base styles - it's not an element styling, it's a new component.

It could fit into `_patterns_image.scss` thematically, but I guess ideal way would be to create a new `_patterns_media-container.scss` file, and treat it as a new componentent (with it's own mixin, etc).